📝 Prepare store description (#74)
lmachens/game-ticker: dist/store/description.txt

# The social network for Gamers! 

Automatically share highlights and comments on your favorite games with your friends. 

## Features

> - Play and let Game Ticker automatically capture your highlights.

Reviewer: ```suggestion
- Play and let Game Ticker take care of the rest.
```

> - Spectate your friends match in a live-ticker. You won't miss any highlight!

Reviewer: ```suggestion
- Spectate your friends match in a live-ticker. You never miss a highlight again!
```

> - Play and let Game Ticker automatically capture your highlights.
> - Spectate your friends match in a live-ticker. You won't miss any highlight!

Reviewer: ```suggestion
- Play and let Game Ticker automatically take care of the rest.
- Watch your friends' matches in the live ticker. You will no longer miss a highlight again!
```
